pubspec.yaml

```diff
@@ -29,7 +29,10 @@ dependencies:
   chewie_audio: ^1.2.0
 
   # Plugins for rendering the <svg> tag.
-  flutter_svg: ^0.21.0-nullsafety.0
+  flutter_svg: ^0.20.0-nullsafety.3
+
+  # Plugin for rendering MathML
+  flutter_math: ^0.3.0-nullsafety.1
```

Comment:
Another external dependency... I should really start work on the modularization.

Reply:
I was watching flutter_widget_from_html and its just completed modularization... It uses widget factories (I still don't know what these are and how they work) and forks of each dependency as an "add-on" to the original lib.